agave_app/renderer.h

```diff
@@ -30,6 +29,58 @@ class IRenderWindow;
 class RenderSettings;
 class Scene;
 
+// serialized so permanent?
+enum eRenderDurationType
+{
+  TIME = 0,
+  SAMPLES = 1
+};
+
+struct RenderDuration
```

Comment:
RenderDuration is a newly introduced data structure to encapsulate the data needed to determine if a render is finished or not.